Framework/Algorithms/src/RadiusSum.cpp

```diff
@@ -227,9 +225,9 @@ void RadiusSum::inputValidationSanityCheck() {
     throw std::invalid_argument(s.str());
   }
 
-  std::vector<double> boundary_limits = getBoundariesOfInputWorkspace();
+  const std::vector<double> boundary_limits = getBoundariesOfInputWorkspace();
   std::stringstream s;
-  BOOST_FOREACH (auto &value, boundary_limits)
+  for (const auto &value : boundary_limits)
     s << value << " , ";
```

Comment:
I think this loop leaves an extra `,` at the end of the string. You may be better of doing something like 

```
std::copy(boundary_limits.begin(), std::prev(boundary_limits.end()), std::ostream_iterator<double>(s, ", "));
s << boundary_limits.back();
```